### DISCUSSION TITLE
Integrate calendar

### DISCUSSION THREAD

**Discussion by vega82** [uyuni-project/uyuni#3365] [Ideas]
Is it possible to integrate a real calendar? 
Ical is very laborious to configure.
It should be better to have a calendar and chose times and dates for a maintenance windows. So for example "repeat every friday 04:00" similar as you can configre in the repository snc - settings.

**Comment by parlt91** [comment]
Hi,
this is actually in the process of being implemented by us at the moment.
For the first iteration we are going to allow users to view their imported maintenance calendars/schedules through a web calendar integrated into the calendar/schedules details view (instead of just presenting the ical files content).
Later on we also want to allow the creation of maintenance calendars/schedules through the Uyuni webUI.
You can find detailed information on what to expect [here](https://github.com/uyuni-project/uyuni-rfc/blob/master/accepted/00075-maintenance-windows-web-calendar.md)
I hope this helps ;)
